### ISSUE TITLE
Relax required_rubygems_version?

### ISSUE THREAD

**Issue by theflow** [indirect/haml-rails#153]
Hi,

Would it be possible to relax the `required_rubygems_version` requirement? Given that Heroku still ships Rubygems 2.6 with Ruby 2.4 and Rubygems 2.7 with Ruby 2.5 makes it a bit hard to use this gem there.

```
remote:        Running: bundle install --without development:test --path vendor/bundle --binstubs vendor/bundle/bin -j4 --deployment
remote:        Bundler Output: Warning: the running version of Bundler (1.15.2) is older than the version that created the lockfile (1.17.3). We suggest you upgrade to the latest version of Bundler by running `gem install bundler`.
remote:        Fetching gem metadata from https://rubygems.org/..............
remote:        Fetching version metadata from https://rubygems.org/..
remote:        Fetching dependency metadata from https://rubygems.org/.
remote:        haml-rails-2.0.0 requires rubygems version >= 3.0.0, which is incompatible with
remote:        the current version, 2.6.14.4
remote: 
remote:  !
remote:  !     Failed to install gems via Bundler.
remote:  !
remote:  !     Push rejected, failed to compile Ruby app.
```

Cheers,
Florian

**Comment by swaincreates** [comment]
Agreed, same with AWS Beanstalk. The [commit](https://github.com/indirect/haml-rails/commit/f9ee1a8815f2099a328e71267a945e365c8a2e7c) that added doesnt describe any technical reason to have it so high.

**Comment by phoet** [comment]
@theflow 👋 just merged a depfu PR that broke our deployment due to this.

haml-rails should get a changelog to warn people of such things. looking into the commit-history is not a feasible solution.

**Comment by ragesoss** [comment]
travis-ci is also still on an incompatible older version of rubygems.

**Comment by holstvoogd** [comment]
Same for semaphoreCI

**Comment by kewubenduben** [comment]
Please do release a relaxed required_rubygems_version.

**Comment by sebastianhegge** [comment]
We are working at our company (i think like many others) with the Ubuntu LTS versions and it's packages. Latest LTS ist 18.04 and it's coming with rubygems version 2.7.6 (2.5.2.1 on 16.04) - so we are actually not able to use haml-rails 2.0.0

**Comment by gstokkink** [comment]
Can a developer please comment on this?

**Comment by lazebny** [comment]
We have the same issue with AWS OpsWorks, Chef 11 and rails 5.

**Comment by j-w** [comment]
Still impacting Heroku.

**Comment by sman591** [comment]
Looks like this was fixed in #154 and released in version 2.0.1, tested with Ruby 2.5 / RubyGems 2.7.6 and the install succeeded

**Comment by amatsuda** [comment]
Right. Fixed via #154.
